File: seungyeop/22_12_22_01.java
class Solution {
    public int solution(int[] common) {
        if (common[1] - common[0] == common[2] - common[1])  //등비인지를 먼저 확인할 경우 나누기 0이 되는 경우가 발생 가능함
            return common[common.length - 1] + common[1] - common[0];
        return common[common.length - 1] * (common[1] / common[0]);
    }
}

File: yejin/22_12_22_01.py
# 옹알이 (1)

def solution(babbling):
    count, max = 0, 0
    a= ["aya", "ye", "woo", "ma"]
    for i in babbling:
        for j in a:
            if j in i:
                count +=1
        if max < count:
            max= count
        count = 0
    return max
  
#   실패
